### PR TITLE
Dont run delocate on libtorch

### PR DESCRIPTION
**Pull request by atalman**
Dont run delocate on libtorch
Fixes: [#74120](https://github.com/pytorch/pytorch/issues/74120)

Testing:
https://github.com/pytorch/pytorch/actions/runs/2447946700

```
otool -L libiomp5.dylib
libiomp5.dylib:
	@rpath/libiomp5.dylib (compatibility version 5.0.0, current version 5.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1225.1.1)
```

### CODE REVIEW

File: wheel/build_wheel.sh [hunk 1]
```diff
@@ -59,6 +59,12 @@ fi
 export PYTORCH_BUILD_VERSION=$build_version
 export PYTORCH_BUILD_NUMBER=$build_number
 
+if [[ -n "$PACKAGE_TYPE" ]]; then
+    package_type="$PACKAGE_TYPE"
+else
+    package_type = 'wheel'
+fi
+
```
Comment (seemethere) [inline wheel/build_wheel.sh:67]:
```suggestion
package_type="${PACKAGE_TYPE:-wheel}"
```

Nit, but we can use bash defaults here

**Review by seemethere — APPROVED** [review]
lgtm just one nit

**Comment by malfet** [comment]
This would break people who install it on Mac, but don't have libiomp.dylib, wouldn't it?

**Comment by atalman** [comment]
I run a comparison with all dylibs included in libtorch that where build with delocate and where delocate step was skipped, and I do see all same dylib, here is visual comparison:
Screen Shot 2022-06-06 at 11 49 33 AM
s

**Pull request merged**